src/components/FormAddMovie/formAddMovie.test.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { FormAddMovie } from "./index";
+import { MemoryRouter } from "react-router-dom";
+import * as Snackbar from "react-simple-snackbar";
+
+describe("Form Add Movie", () => {
+  const openSnackbarMock = jest.fn();
+  const closeSnackbarMock = jest.fn();
+  jest
+    .spyOn(Snackbar, "useSnackbar")
+    .mockImplementation(() => [openSnackbarMock, closeSnackbarMock]);
+  it("renders appropriately", () => {
+    render(
+      <MemoryRouter>
+        <FormAddMovie />
+      </MemoryRouter>
+    );
+    expect(screen.findAllByDisplayValue("Submit"));
+  });
+
+  it("show the sended message", () => {
```

Comment:
Consider replicating this test structure in other test where applicable, so the body is AAA (Arrange, Act, Assert). It helps to have more readable test